packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -1,26 +1,34 @@
 /* eslint-disable max-classes-per-file */
+import EventNames from '../event-names';
+
 class CreateGuestUserError extends Error {
     constructor (message) {
         super(message);
-        this.message = message;
+        this.messageKey = 'errorMessages.guestUserCreationFailure';
```

Comment:
Rather than the key, we could be assigning the message directly. What do you think?

Reply:
Then you would have to inject the translator into the exception class to resolve the translation, didn't really want that responsibility on the simple class, also unsure what we gain.

Reply:
Yup, fair enough.